lab_3/main.py

```diff
@@ -29,23 +55,32 @@ class LetterStorage:
     Stores and manages letters
     """
 
-    def __init__(self):
+    def __init__(self, count=0):
```

Comment:
Please, do not alter the method definitions. Define the `count` inside the constructor and set the default value.

Reply:
done